### ISSUE TITLE
[HELP] Meaning of full_peaks option

### ISSUE THREAD

**Issue by imankahbasi** [eqcorrscan/EQcorrscan#395]
# What do you need help with?
I don't understand the difference between True or False of `full_peaks` option in `find_peaks2_short` function exactly, I think it needs more descriptions and examples.

Furthermore in `match_filter` function and `Tribe().construct()` full_peaks is set to False. But the description in the `find_peaks2_short` function says that this option is set to True in match_filter by default.
Which one is True?

**Comment by calum-chamberlain** [comment]
It looks like when I fixed the issue that `full_peaks` was there for in #249, I didn't update the docs properly. Because `match_filter` uses the compiled version of find_peaks, then the default was switched to `False`. I have updated the docs on master to reflect this.

`full_peaks=True` uses a more exhaustive method of finding peaks which is not necessary for data that cross the threshold frequently.

More docs would be helpful, but this is a really low priority for me.
